### ISSUE TITLE
Clean build failed

### ISSUE THREAD

**Issue by anelrh26** [octo-technology/sonar-objective-c#77]
When I run sh ./run-sonar.sh -v in my root project, I get this

Scheme Tests is not currently configured for the clean action.

*\* CLEAN FAILED *\* (3984 ms)

This is a project that didn't have tests, so I added a scheme for tests in Xcode, but I have the same results

If I run the following command I get more info about the error

xctool -workspace ProjectName.xcworkspace -sdk iphonesimulator ARCHS=i386 VALID_ARCHS=i386 CURRENT_ARCH=i386 ONLY_ACTIVE_ARCH=NO -schemeProjectName clean -showTasks

---

  xcodebuild build clean
# 

LAUNCHING TASK (running xcodebuild):

  TERM_SESSION_ID=A7B63C83-56F5-4E10-A30F-A75E93B607C2 \
  SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.yv7t7UM76h/Listeners \
  OLDPWD=/Users/peter/ProjectName/Repository \
  Apple_PubSub_Socket_Render=/private/tmp/com.apple.launchd.BSyjCLpyts/Render \
  XPC_FLAGS=0x0 \
  DYLD_INSERT_LIBRARIES=/usr/local/Cellar/xctool/0.2.2/libexec/lib/xcodebuild-shim.dylib \
  XT_DID_SET_DYLD_FALLBACK_FRAMEWORK_PATH=YES \
  PWD=/Users/peter/ProjectName/Repository/iphone \
  SHELL=/bin/bash \
  LC_CTYPE=UTF-8 \
  TERM_PROGRAM_VERSION=343.6 \
  TERM_PROGRAM=Apple_Terminal \
  PATH=/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/oclint-0.8.1/bin:/usr/local/git/bin:/Users/peter/Desktop/adb-fastboot-mac:/usr/local/mysql/bin \
  SHLVL=1 \
  TERM=xterm-256color \
  _=/usr/local/bin/xctool \
  HOME=/Users/peter \
  TMPDIR=/var/folders/63/xtt0njvx5fd4n8cc6wblk8_m0000gn/T/ \
  USER=peter \
  XPC_SERVICE_NAME=0 \
  LOGNAME=peter \
  __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x0 \
  DYLD_FALLBACK_FRAMEWORK_PATH=:/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/PrivateFrameworks:/Applications/Xcode.app/Contents/Developer/Library/PrivateFrameworks:/Applications/Xcode.app/Contents/Developer/../OtherFrameworks:/Applications/Xcode.app/Contents/Developer/../SharedFrameworks \
  /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild \
    -configuration \
    Debug \
    -sdk \
    iphonesimulator8.1 \
    PLATFORM_NAME=iphonesimulator \
    ARCHS=i386 \
    CURRENT_ARCH=i386 \
    ONLY_ACTIVE_ARCH=NO \
    VALID_ARCHS=i386 \
    -workspace \
    /var/folders/63/xtt0njvx5fd4n8cc6wblk8_m0000gn/T/xctool_temp_JDUeEt/Tests.xcworkspace \
    -scheme \
    Tests \
    OBJROOT=/Users/peter/Library/Developer/Xcode/DerivedData/ProjectName-finaefweyeteigefwkogjznqykem/Build/Intermediates \
    SYMROOT=/Users/peter/Library/Developer/Xcode/DerivedData/ProjectName-finaefweyeteigefwkogjznqykem/Build/Products \
    SHARED_PRECOMPS_DIR=/Users/peter/Library/Developer/Xcode/DerivedData/ProjectName-finaefweyeteigefwkogjznqykem/Build/Intermediates/PrecompiledHeaders \
    -IDECustomDerivedDataLocation=/var/folders/63/xtt0njvx5fd4n8cc6wblk8_m0000gn/T/xctool_temp_JDUeEt/DerivedData \
#     clean

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Scheme Tests is not currently configured for the clean action.

**Comment by cyrilpicat** [comment]
there seems to be an issue with your test scheme again.

Instead of fixing it, have you tried disabling tests in the properties file?

<pre>
# Comment following line of you don't have any tests
sonar.objectivec.testScheme=MyProjectTests
</pre>

**Comment by anelrh26** [comment]
Yes I have disabled the tests in the properties files like you're suggesting. And I have the same result.
